helm-charts/falcon-sensor/values.yaml

```diff
@@ -81,7 +81,37 @@ container:
   enabled: false
 
   # Configure the number of replicas for the mutating webhook backend
-  replicas: 1
+  replicas: 2
+
+  affinity:
+    # Setting Pod anti-affinity soft or preferred rule so that replicas will try to run on different nodes
+    podAntiAffinity:
```

Comment:
Just mentioning `topologySpreadConstraints` in case they haven't been considered.
The same result can be achieved with something like
```
topologySpreadConstraints:
- maxSkew: 1
  topologyKey: kubernetes.io/hostname
  whenUnsatisfiable: ScheduleAnyway
  labelSelector:
    matchLabels:
      crowdstrike.com/component: crowdstrike-falcon-injector
```
topologySpreadConstraints are a little more flexible than anti-affinity rules, there is a good explanation [here](https://kubernetes.io/blog/2020/05/introducing-podtopologyspread/)

Reply:
Implemented it using topologySpreadConstraints after confirming about supported K8s version